controllers/payment.controllers.js

```diff
@@ -92,12 +92,14 @@ async function newSetupIntent(req, res, next) {
       payment_method_types: ['card'],
     });
   } catch (err) {
+    logger.error(err)
     const error = new HttpError(
       'Could not save card. Please try again later',
       500,
     );
     return next(error);
   }
+  logger.info('The card was added successfully')
```

Comment:
Could you please add info about which user created a card?

Reply:
Sorry that is not possible because we don't receive card information on the server, it goes to Stripe

Reply:
Done, check please